Source/MLX/Factory.swift

```diff
@@ -625,9 +683,9 @@ public func linspace<T: HasDType>(
 ///
 /// ### See Also
 /// - <doc:initialization>
-/// - ``repeat(_:count:stream:)``
+/// - ``repeated(_:count:stream:)``
 /// - ``full(_:values:stream:)``
-public func `repeat`(_ array: MLXArray, count: Int, axis: Int, stream: StreamOrDevice = .default)
+public func repeated(_ array: MLXArray, count: Int, axis: Int, stream: StreamOrDevice = .default)
```

Comment:
This is consistent with the swift naming style AND not a keyword.  E.g. flatten -> flattened (since these are not mutating).  So repeat -> repeated.